imnodes.cpp

```diff
@@ -1772,6 +1778,15 @@ void EndNode()
     g.canvas_draw_list->ChannelsMerge();
 }
 
+ImVec2 GetNodeDimensions(int node_id)
+{
+    EditorContext& editor = editor_context_get();
+	const int node_idx = editor.nodes.find(node_id);
+    assert(node_id != -1); //invalid node_id
+	const NodeData& node = editor.nodes.pool[node_idx];
+    return ImVec2(node.rect.GetWidth(), node.rect.GetHeight());
```

Comment:
Actually, you could just return `node.rect.GetSize()` which does the same thing

Reply:
indeed better, fixed in 7a217e5